src/module/game/guess_1a2b.py

```
@@ -0,0 +1,135 @@
+    current_info: GuessInfo = _guess_info[current_uuid]
+
+    if current_info.status == GuessStatus.RUNNING:
+        message.reply("游戏已经开始，请使用 \"/1a2b [num]\" 猜测数字，不要带上中括号\n\n"
+                      f"目标为 {len(current_info.target)} 位数")
+        return None
+
+    target_len = random.randint(4, 7)
+    target = ''.join(random.sample("0123456789", k=target_len))
+    if target[0] == '0':  # 避免出现前导0
+        rnd_idx = random.randint(1, len(target) - 1)
+        target[0], target[rnd_idx] = target[rnd_idx], target[0]
```

Pylint (3.3.7)

[error] 'target' does not support item assignment

(E1137)